src/libpmemfile-posix/data.c

```diff
@@ -87,10 +87,17 @@ narrow_to_full_pages(uint64_t *offset, uint64_t *length)
 /*
  * block_cache_insert_block -- inserts block into the tree
  */
-static void
+static int
 block_cache_insert_block(struct ctree *c, struct pmemfile_block *block)
```

Comment:
BTW, perhaps we should mark such functions with the warn_unused_result GCC attribute.
For example ctree_insert_unlocked could be marked.